reconciler/retry.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2020 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Can you please use the standard license header.
We have this botched one in a few places, but we should not introduce new ones.
https://github.com/knative/pkg/blob/bec759b1d79ec5e1d2c00450f04477e85b7dc3fc/reconciler/events.go, e.g. -- see empty lines around the URL.

Reply:
fixed